app/models/review.rb

```diff
@@ -0,0 +1,14 @@
+class Review < ApplicationRecord
+  belongs_to :art
+  belongs_to :user
+
+  validates :rating, presence: true, inclusion: 1..5, numericality: { only_integer: true }
+  validate :user_has_booked_before
```

Comment:
You need to make another validation to confirm the user has not reviewed that art already